generate_pdbbind_data.py

```diff
@@ -35,7 +40,32 @@ def extract_labels(pdbbind_label_file):
       labels[line[0]] = line[3]
   return labels
 
-def generate_dataset(pdbbind_label_file, feature_files, out_csv):
+def write_csv(feature_dict, labels, out):
+  """Write CSV output."""
+  with open(out, "wb") as csvfile:
+    writer = csv.writer(csvfile, delimiter="\t")
+    writer.writerow(["Smiles", "Sequence", "Label", "Features"])
+    for key in feature_dict:
+      label = labels[key]
+      (features, smiles, seq) = feature_dict[key]
+      writer.writerow([smiles, ",".join(seq), label, ",".join([str(elem) for elem in list(features)])])
```

Comment:
it might be nice to decompose this line a little bit, i.e. store ",".join([str(elem).....]) as a separate variable before the row-writing line